tests/dummy/app/components/font-weight-line.js

```diff
@@ -0,0 +1,5 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+	classNames: 'typescale-line'
```

Comment:
tagName: ''
and add the wrapping div to template
cleaner approach
one that will be soon default